ports/qtwebengine/vcpkg.json

```
@@ -57,6 +58,9 @@
     },
     {
       "name": "libxml2",
+      "features": [
```

```suggestion
      "default-features": false,
      "features": [
```
I don't which features are really required in addition to `icu`. But opting-out must be requested explicitly anyways, so it can be allowed without hurting the default use case.

---

Thanks for your advice.